src/plugins/intel_gpu/src/graph/primitive_inst.cpp

```diff
@@ -649,6 +651,14 @@ bool primitive_inst::has_inner_networks() const {
     return (_impl_params->inner_nets.size() > 0);
 }
 
+bool primitive_inst::use_async_compilation() const {
```

Comment:
I think this need not be a public member function of primitive_inst. Could  you add this as a lambda function above?

Reply:
applied as comment